docs/source/author/topics.rst

```diff
@@ -41,19 +41,16 @@ diagram.
 
 This is just a broad overview of the topics application.
 
-
-.. _viewing-the-current-state:
-
-Viewing the Current State
+Viewing All Topics Content
 ------------------------------------------------------------------------------
 
-When developing locally, once you have a server and ``gulp`` running (see
-:doc:`../getting_started/basic_usage`) you can go to the url below to get a
+When developing locally, once you have run ``csu start`` (see
```

Comment:
there is quite a bit of inconsistency here. We run `./csu start` but throughout the whole docs we don't prefix commands with the `./`, i.e. `csu` - this doesn't even work on my machine, I have to have the `./` there, so I think we should include this in all the docs rather than just mention it once at the start, what do you think?